src/data/doc-extensions/extensions.json

```diff
@@ -217,7 +218,8 @@
         "url": "https://github.com/dgzlopes"
       },
       "official": false,
-      "categories": ["Misc"]
+      "categories": ["Misc"],
+      "type": "Out"
```

Comment:
This is not actually an output extension. For purposes of this PR, we can remove all uses of the "type" attributes for now given we see it is available.

Reply:
Yes, i will remove all uses of the `type` attributes in `extensions.json`.